### ISSUE TITLE
Colorization becomes invalid/stale after brace completion in string interpolation 

### ISSUE THREAD

**Issue by NinoFloris** [JetBrains/resharper-fsharp#263]
Colorization becomes invalid/stale after brace completion in string interpolation 

```fsharp
let test = async {
    return $"Add an interpolation hole here: "
} // after interpolation hole is inserted this bracket (and all following in the document is now string colored until file close and reopen)
```

As shown in the screenshot 
<img width="646" alt="Screenshot 2021-05-14 at 18 59 48" src="https://user-images.githubusercontent.com/4218809/118304185-a34ad400-b4e6-11eb-8c64-bed8982e3019.png">

**Comment by auduchinok** [comment]
@NinoFloris Thanks for reporting, I'm able to reproduce it and will look into it soon.

**Comment by auduchinok** [comment]
It's fixed in upcoming 2021.2 EAP. Maybe we'll include the fix in some 2021.1 update too, but not sure yet.
